app/api/scripts/util/script_util.py

```diff
@@ -1,8 +1,9 @@
+# TODO use built in @flask.cli commands so that we can reuse flask app and no longer need this file
```

Comment:
I'm not familiar with Flask CLI, but this is explicitly for backend scripts separate from the API itself, like a lot of our backend processes from PFML (payments, file imports, and so on).

Reply:
Kind of related to the [other thread](https://github.com/navapbc/template-application-flask/pull/74#discussion_r1068634113) so maybe we can just continue on that thread, but I was surmising that Flask custom CLI commands could replace our custom/homegrown entrypoints for backend processes.